services/worker-manager/test/fixtures/azure_signature_good.json

```diff
@@ -0,0 +1,7 @@
pOoownR5x11zJjT+1ISr0Gj77SE8ZNJyTc9tb93g+Bj4+M0sFETgLSmJ2d4QlFjpzPwEswKhqk9FSCSmbagadUhS6N1XzspeXbbkIdS1YQDC3beZSt8wbwkj9RvCkc5fz4ScRlKuFGzKCYgjKZrR6tDwPkSTIaGF+KiIErxxndyU8biM+ctRlPF71szPS/0lVaEQkbCMM8paufX2gilwwqYYnSjhfVx3IrjBbjDqHzZhbU6He5O6ze4ZnjEJUgaKmVHxQE2lJuwSQgeg1piXCv8hiayuJTCQIDAQABo4IEMDCCBCwwggF/BgorBgEEAdZ5AgQCBIIBbwSCAWsBaQB3AN3cyjSV1+EWBeeVMvrHn/g9HFDf2wA6FBJ2Ciysu8gqAAABlSAWl54AAAQDAEgwRgIhAK7co4XBXyJtU5UoNqpNmvKo8aJJmEBEos7iSO5IYsI3AiEA+6kFycUs2XC6uDt0cQauypkKRxHOwUiu4RzRUdaOUuMAdgB9WR4S4XgqexxhZ3xe/fjQh1wUoE6VnrkDL9kOjC55uAAAAZUgFpdQAAAEAwBHMEUCIQDLwm81zOPKvwTseDbduDyH4ilr/2bOaaL9pPdPQweo7AIgYu8kntVejISi1dOv6KnI/NYm0ziivdGmxw2Tpz6MFyAAdgAaBP9J0FQdQK/2oMO/8djEZy9O7O4jQGiYaxdALtyJfQAAAZUgFpfMAAAEAwBHMEUCIQCWmipbKkFUJ4PtBrWUyh236z7ER7slrPCoPa7l3bHgOAIgWlNKFarSRDI+rdLuFXZ2/58boos6OHTScUErrqYdAocwJwYJKwYBBAGCNxUKBBowGDAKBggrBgEFBQcDAjAKBggrBgEFBQcDATA8BgkrBgEEAYI3FQcELzAtBiUrBgEEAYI3FQiHvdcbgefrRoKBnS6O0AyH8NodXYKr5zCH7fEfAgFkAgEtMIG0BggrBgEFBQcBAQSBpzCBpDBzBggrBgEFBQcwAoZnaHR0cDovL3d3dy5taWNyb3NvZnQuY29tL3BraW9wcy9jZXJ0cy9NaWNyb3NvZnQlMjBBenVyZSUyMFJTQSUyMFRMUyUyMElzc3VpbmclMjBDQSUyMDAzJTIwLSUyMHhzaWduLmNydDAtBggrBgEFBQcwAYYhaHR0cDovL29uZW9jc3AubWljcm9zb2Z0LmNvbS9vY3NwMB0GA1UdDgQWBBSU/hX8bXzQmKwu4Q8R/LOQ8xjMgDAOBgNVHQ8BAf8EBAMCBaAwOAYDVR0RBDEwL4IZZWFzdHVzLm1ldGFkYXRhLmF6dXJlLmNvbYISbWV0YWRhdGEuYXp1cmUuY29tMAwGA1UdEwEB/wQCMAAwagYDVR0fBGMwYTBfoF2gW4ZZaHR0cDovL3d3dy5taWNyb3NvZnQuY29tL3BraW9wcy9jcmwvTWljcm9zb2Z0JTIwQXp1cmUlMjBSU0ElMjBUTFMlMjBJc3N1aW5nJTIwQ0ElMjAwMy5jcmwwZgYDVR0gBF8wXTBRBgwrBgEEAYI3TIN9AQEwQTA/BggrBgEFBQcCARYzaHR0cDovL3d3dy5taWNyb3NvZnQuY29tL3BraW9wcy9Eb2NzL1JlcG9zaXRvcnkuaHRtMAgGBmeBDAECAjAfBgNVHSMEGDAWgBT+CXFAVQUQRNikgXW4nhrpSgaIyDAdBgNVHSUEFjAUBggrBgEFBQcDAgYIKwYBBQUHAwEwDQYJKoZIhvcNAQEMBQADggIBAFx/SFHgfTktPlQxZiaMJHCjOqmvWp7wBGtUIUfbfuEfE85Fbbv1pNdhWnuw3mV5X42w8Y5b9YpDoWY3BApqfxCqyEvMxNx/eM9+Qmdn9c4shGD9G5bxkfhsjDR63jt8+fV6psNyLB3Pe2nIo/vKmKcm/tvkvsvkMTY/JRLfE8VWQghGyCnUOM+V/hX57o4ZJy73Lb0rVxFA9SsEWXEZ8mcIOQkV5ZAOJl9u/wXZ+AJgbqYzLwBIgqnvtcebU5aw9jswoPBF1Ftgw8Sat7H6bMRClJha7WUKlsTqtDrZP2eyJWyMYUmVdTXwIQLLeXQICusgG80P4Lu0uhITRE/QEU3T8sdFNtMyMR7jmM24eEko5JhTFyxhzaCNC1iN2ubBMJkZ6SX+hVtzzk3Wah1yi8LoaIkyT/hHZYHTidWgaZCgaCG4emTOoD8r6A+TbXxC9gFyo2wHNfZsuqIZShxixuNeNHk8em7X7S1RRq0vY/RijDYcdZWK2IadmaMeE2fXxnGPHa9kLJrO0X7vuV3b1bRmLYGlqDITInRj+iaURwIDuNgi+m4UHz+oLyKnqQmwM6/cOuBG0tpZMtik9eCbhO4I3hQtY62JS3lI4H+Xfq22TEO5L0rJUvcksY+e7+vBjqx55GqbnwfnTXgwdH2A8LpxIDqufZZFlAuxiFZQx0WGMYIBnzCCAZsCAQEwdDBdMQswCQYDVQQGEwJVUzEeMBwGA1UEChMVTWljcm9zb2Z0IENvcnBvcmF0aW9uMS4wLAYDVQQDEyVNaWNyb3NvZnQgQXp1cmUgUlNBIFRMUyBJc3N1aW5nIENBIDAzAhMzAcsTK05ojcNkJTLGAAAByxMrMA0GCSqGSIb3DQEBCwUAMA0GCSqGSIb3DQEBAQUABIIBAFpe9LfcwbXphl8/BJXOY+gdQICgPsvLtR+ctWhI/PPjeSVCON3LY/TwmG5XirRMngYk0hlFJ87mPUQShpD6mhLFnvtpx1n/MxKPUqO1tDgvNKYoUDp1I0wjfz182h2k/Vu+YCB55Vjns/0pgo2XCZfvg4fo7nLJf1zB6uaCleqNLdzQJyAMK/xWaGHfVIzRfiBj8RpcfedFB3p7jtlBBCPV5EIkCpXNgRQglWWCDfW7NYIrNuZiNwN5L0oh+fW8xlikpi+QGsLjRbo3UTItE+5unTxwzw8aD7yf+eoUk3fZOXDRq4bzaM7KkHyqx9gKqG0+hXJJq32a/vAqt6QbxKw=",
```

Comment:
moved good signature and vmId to a single file, so only this needs to be updated when cert expires